packages/squiggle-lang/src/rescript/Reducer/Reducer_ErrorValue.res

```diff
@@ -1,15 +1,22 @@
 type errorValue =
   | REArrayIndexNotFound(string, int)
+  | REAssignmentExpected
```

Comment:
Are we sure we want to call this `Assignment` rather than `Statement`? I think the bottom line of most squiggle files, where a chart is rendered, is better thought of as a statement than an expression. unless it's thought of as an effectful expression. Idk I'm uncertain.

Reply:
@quinn-dougherty : My impression is that this is used in the very specific case where the parser took there to be a "$$bindStatement" without a ``let`` in it. I'd guess that this would be near impossible, but maybe there's some way for it to happen.

Reply:
All expressions return a result even if it is  void or haskell's (). Here I am really meaning that it has to be an assignment